_data/translations.yml

```diff
@@ -150,3 +150,18 @@ de:
     order-poll: "Diagramm welches zeigt wie Entwickler ihre CSS Deklarationen ordnen"
     sassdoc: "Dokumentation von SassDoc generiert"
     wallpaper: "Wallpaper von [Julien He](https://twitter.com/julien_he)"
+
+da:
+  donate:
+    content: "Hvis du nød Sass Guidelines, overvej venligst at støtte dem."
+    button: "Støt Sass Guidelines"
+  baseline:
+    content: "En subjektiv styleguide til at skrive fornuftigt, vedligeholdelsesvenligt og skalerbart Sass."
+  footer:
+    content: "Lavet med kærlighed af [Hugo Giraudel](http://hugogiraudel.com)"
```

Comment:
Please, add your name here so that it reads something like *Made with love by [Hugo Giraudel](http://hugogiraudel.com), translated by [Anders Schmidt Hansen](http://codepen.io/andersschmidt/)*.